Feature/repair build

JanDotNet opened the pull request.

Peter-B- commented:
Hi Jan-Niklas,

thanks for your effort. I compiled the nuget package and tested it with my .Net 5 application and it works. It even worked with the license file I created with .Net 4.6., so it seems the hardware identifiers still work. It didn't accept the license file on another computer.

One thing though: During my update, I moved `RsaExtensions` from `ThinkSharp.Licensing` to `ThinkSharp.Licensing.Signing.RSA`. I think you should revert this, since it is an unnecessary breaking change for the `WithRsaPrivateKey` extension method.

JanDotNet commented:
Hi Peter,

thanks again for your improvements. I changed the namespace of RsaExtensions back to 'ThinkSharp.Licensing'.  The new packages has been published on nuget.org.

Further more I added you/your contribuition in the 'Credits' section within the readme.md file. Hope thats ok for you :).

Peter-B- commented:
Thanks for publishing it.

And thanks you for mentioning me. That's a nice gesture.
